webapp/app/templates/basis/base.html

```diff
@@ -7,22 +7,25 @@
     <meta name="twitter:card" content="summary_large_image">
     <meta name="twitter:title" content="{{ _('meta.og.title') }}">
     <meta name="twitter:description" content="{{ _('meta.og.description') }}">
-    <meta name="twitter:image" content="https://www.steuerlotse-rente.de/static/images/STL_og_image.jpg">
+    <meta name="twitter:image" content="https://www.steuerlotse-rente.de{{ static_url_for('static', filename='/images/STL_og_image.jpg') }}">
```

Comment:
We can't just use `static_url_for` because it returns a relative path and we need an absolute path here, correct?

Reply:
I'm actually not sure why we were using hard-coded URLs here. I vaguely recall something about being able to test Twitter cards on staging? We _could_ use `static_url_for` with `_external=True` to generate a link that includes the host and scheme.

Reply:
Okay, we do not have to change it. I was just wondering, why we are using the hard-coded URLs. Would make sense to enable testing of Twittercards.